doc/diagnostics/esmvaltool.rst

```diff
@@ -26,20 +26,26 @@ An example of steps to run the ESMValTool on ipcc.nird.sigma2.no
 
 2. load ESMValTool on nird: ::
 
-    conda active /conda/esmvaltool/2.0.0b4/
+    conda active /conda/esmvaltool/2.0.0/
 
 (note, the esmvaltool may be upgraded in the future. Therefore, use ``ls /conda/esmvaltool/`` to check the currently installed vesion if 2.0.0b4 does not exist.)
 
-3. copy the following config file and recipe to your favourite place: ::
+3a  run initialisation : ::
+    esmvaltool config get_config_user
 
-    /projects/NS9252K/share/yanchun/esmvaltool/config_heyc.yml
-    /projects/NS9252K/share/yanchun/esmvaltool/recipe_seaice.yml
-    
-You should copy and modify the config_heyc.yml file to store some intermediate data files and final plots to your own directory.
+3b. copy the following config file to ~/.esmvaltool/::
+    /projects/NS9252K/share/esmvaltool/config/config-ipcc.yml
+    /projects/NS9252K/share/esmvaltool/config/config-developer.yml
 
+You should copy and modify the config-ipcc.yml file to store some intermediate data files and final plots to your own directory.
+e.g. change path of config-developer.yml in config-ipcc.yml
+ 
+3c. copy the following recipe to ~/your-recipes/::
+    /projects/NS9252K/share/esmvaltool/config/recipe_validation_CMIP6.yml
+    
 4. run ``esmvaltool``: ::
 
-    esmvaltool -c config_xxx.yml ./recipe_seaice.yml
+    esmvaltool run --config_file .esmvaltool/config-ipcc.yml ./your-recipes/recipe_validation_CMIP6.yml
```

Comment:
I find the `config_ipcc.yml` can not find the `config_developer.yml` in line:
```
config_developer_file: ./config-developer.yml
```
So I update the line in `config_ipcc.yml` to:
```
config_developer_file: ~/.esmvaltool/config-developer.yml
```

Just curious if this command really works before this change?

And please test at your side if this works after the changes.

However, this  has no impact on the document here.